packages/cli/src/commands/mock.ts

```diff
@@ -10,8 +12,13 @@ const mockCommand: CommandModule = {
         description: 'Path to a spec file. Can be both a file or a fetchable resource on the web.',
         type: 'string',
       })
-      .fail(() => {
-        yargs.showHelp();
+      .middleware(async argv => {
+        argv.operations = await getHttpOperations(argv.spec!);
+      })
+      .fail((msg, err) => {
+        if (msg) yargs.showHelp();
```

Comment:
This is basically to make sure that it will show help in case you're missing the argument (as @philsturgeon asked) and forward the error in case it's happening for any other reason (a thrown exception or something)